styles/blog.module.css

```diff
@@ -0,0 +1,8 @@
+.blogsList {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
Would love to see if you could do 3 cards per row with flexbox and max-width on the container, and then it responsively adjusts to 2 and then 1 column.

Reply:
I think I can do this.